lib/grovepi/rgblcd.ex

```diff
@@ -430,15 +442,18 @@ defmodule GrovePi.RGBLCD do
     send_chars(rest)
   end
 
-  defp send_lcd_cmd(cmd) do
+  @doc false
```

Comment:
Instead of moving these to public in this module what if we create a new module? Or two modules? One for LCD and one for RGB. It seems to make sense since we have multiple methods that you want a test around and they have a similar name. To me the fact that two of them start with LCD points to us needing an LCD module. The new module would also put a place for us to hang each address.  The display that we are using is two components, so it seems to make sense to me.

Reply:
The RGB module would just have `set_rgb`, `set_color_white`, and `send_rgb`.  Would you leave `initialize` in LCD?  It will then call into the RGB module.  I think the RGB functions might get lost in their own module.  

I'm not sure I understand what you mean by the display having two components.  The LCD commands control text on the screen and the RGB commands control the color of the screen.

I think it might be easier for users of the library to only need to reference docs in a single module when working with the display.

If you think the two modules will make everything easier to understand and maintain, I can definitely break them apart into two modules.  Let me know what you think.